### ISSUE TITLE
prismacloudcompute_container_runtime_policy provision fails with 400 code in PCCE 30.00.140

### ISSUE THREAD

**Issue by adrianfuro** [PaloAltoNetworks/terraform-provider-prismacloudcompute#72]
I built a Terraform module for Runtime using dynamic blocks and I am trying to provision the rules, but I get the following error:

```
Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

module.runtime.prismacloudcompute_container_runtime_policy.ruleset: Creating...
╷
│ Error: error creating containerRuntime policy: Non-OK status: 400
│
│   with module.runtime.prismacloudcompute_container_runtime_policy.ruleset,
│   on modules\runtime\main.tf line 11, in resource "prismacloudcompute_container_runtime_policy" "ruleset":
│   11: resource "prismacloudcompute_container_runtime_policy" "ruleset" {
│

```

Another thing to add, this module was working fine before PCCE upgrade from 22.06 to current version. I also tried to provision other resources such as **prismacloudcompute_container_compliance_policy** and is working perfectly fine on PCCE 30.00.140

**Comment by maxboynton** [comment]
Issue has been addressed in the v0.8.0 release (https://github.com/PaloAltoNetworks/terraform-provider-prismacloudcompute/pull/71).

**Comment by adrianfuro** [comment]
Good to know, thanks!
